prow/cmd/artifact-uploader/BUILD.bazel

```diff
@@ -18,6 +18,18 @@ go_library(
     ],
 )
 
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+
+go_image(
+    name = "image",
+    base = "@alpine-base//image",
+    embed = [":go_default_library"],
+    goarch = "amd64",
+    goos = "linux",
+    pure = "on",
+    visibility = ["//visibility:public"],
+)
```

Comment:
artifact-uploader previously did not have a bazel-defined image

Reply:
@stevekuznetsov what would we use an image for with this one? this:

https://github.com/kubernetes/test-infra/blob/8252d8fda5100fab4ce69ed0f9d303fcb7e1d567/prow/config.yaml#L8-L12

seems to confirm that we do not use this (the sidecar instead)?